<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="input_search"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Указать файл...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="mybrowser" name="output">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_ID">
        <property name="minimumSize">
         <size>
          <width>99</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Название предмета</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_subj"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Аудитория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_aud"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Тип</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="box_type">
        <item>
         <property name="text">
          <string>Lesson</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Examenation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lab</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1028</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mybrowser</class>
   <extends>QTextBrowser</extends>
   <header>mybrowser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchButton</tabstop>
  <tabstop>input_ID</tabstop>
  <tabstop>input_aud</tabstop>
  <tabstop>input_subj</tabstop>
  <tabstop>button_add</tabstop>
  <tabstop>output</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchLes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>input_search</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>898</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addLes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1015</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add</sender>
   <signal>clicked()</signal>
   <receiver>input_aud</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1015</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>770</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add</sender>
   <signal>clicked()</signal>
   <receiver>input_subj</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1015</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add</sender>
   <signal>clicked()</signal>
   <receiver>input_ID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1015</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>998</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>1021</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchLes()</slot>
  <slot>addLes()</slot>
  <slot>showAll()</slot>
  <slot>browse()</slot>
 </slots>
</ui>
